### ISSUE TITLE
Please make the toggle switch accessible

### ISSUE THREAD

**Issue by assem-ali-tfnsw** [tableau/extension-single-checkbox-parameters#41]
Currently the toggle switch cannot be controlled using just the keyboard. When I added it to my dashboard, I couldn't navigate to it using the tab key on the keyboard. This makes not compliant with accessibility standards and not suitable for users that cannot use the mouse and rely on screen readers to navigate through the dashboard.
Please ensure a) user can navigate to the control using tab, and b) users are able to toggle the switch using spacebar.
